gwdetchar/lasso/core.py

```diff
@@ -33,61 +33,91 @@
 
 # -- utilities ----------------------------------------------------------------
 
-def find_outliers(ts, N=5):
+def find_outliers(ts, L=5, method='s'):
     """Find outliers within a `TimeSeries`
 
     Parameters
     ----------
     ts : `~gwpy.timeseries.TimeSeries`
         data to find outliers within
 
-    N : `float`, optional
-        number of standard deviations to consider an outlier, default: 5
+    L : `float`, optional
+        if S is given as 's': number of standard deviations to consider an outlier, default: 5
```

Comment:
Based on the kwarg names in the function definition, this should probably read something like:

```suggestion
        if ``method='s'``: number of standard deviations to consider an outlier, default: 5
```

etc.

Note: These comments apply to all changed function definitions and docstrings